src/app/code/community/EasyTranslate/Connector/Model/Resource/Project.php

```diff
@@ -290,4 +290,14 @@ protected function _lookupTargetStoreIds(int $id): array
 
         return $adapter->fetchCol($select, $binds);
     }
+
+    private function randomiseString()
```

Comment:
Please add a return type.

`randomise` sounds like we randomise a specific string. Instead, we create / return a random string. So maybe better use `getRandomString` or `createRandomString`.